demisto_sdk/commands/init/initiator.py

```diff
@@ -737,19 +1076,28 @@ def yml_reformatting(self, current_suffix: str, integration: bool = False):
         if from_version:
             yml_dict["fromversion"] = from_version
 
-        if Version(
+        if not self.xsiam and Version(
             yml_dict.get("fromversion", DEFAULT_CONTENT_ITEM_FROM_VERSION)
         ) < Version(self.SUPPORTED_FROM_VERSION):
             yml_dict["fromversion"] = self.SUPPORTED_FROM_VERSION
 
+        elif self.xsiam and Version(
+            yml_dict.get("fromversion", DEFAULT_CONTENT_ITEM_FROM_VERSION)
+        ) < Version(self.SUPPORTED_FROM_VERSION_XSIAM):
+            yml_dict["fromversion"] = self.SUPPORTED_FROM_VERSION_XSIAM
```

Comment:
duplicate code - please replace the compared version (`self.SUPPORTED_FROM_VERSION_XSIAM` / `self.SUPPORTED_FROM_VERSION`) based on `if self.is_xsiam`, instead of duplicating code